assets/shaders/compositor.frag

```diff
@@ -32,9 +32,11 @@ struct PipelineData {
     int useSSAO;
     int useOutline;
     int useHDRI;
-    int useVolume;
+    int useToneMap;
 
+    int useVolume;
     vec3 volumeColor;
+
     float density;
```

Comment:
pading?

Reply:
does the last element need padding

Reply:
it will be better to prevent forget add padding in the future

Reply:
ok